addons/ai/fnc_searchNearby.sqf

```diff
@@ -59,13 +65,18 @@ if ((leader _group) distanceSqr _building > 250e3) exitwith {};
 
         // Send all available units to the next available position
         {
+            private _starttime = _x getVariable _timetag;
+
             if (_positions isEqualTo []) exitWith {};
-            if (unitReady _x) then {
+            if (unitReady _x || {_starttime + _timeout < time}) then {
```

Comment:
If lazy eval is used here, may as well inline the whole local variable.

Reply:
As such?
```
if (unitReady _x || {(_x getVariable _timetag) + _timeout < time}) then {...}
```